html_reference.html

```diff
@@ -19,7 +20,7 @@
 					<a><i class="fas fa-share-alt-square"></i></a>
 				</div>
 			</div>
-			<h1 class="myName">milo_girardet  <i class="fas fa-user-secret" id="astronaut"></i></h1>					
+			<h1 class="myName">milo_girardet  <i class="fas fa-user-astronaut" id="astronaut"></i></h1>					
```

Comment:
Your class naming is alternating between two standards - camelCase and lowercase-separated-by-dashes.
Either of these is fine on their own, but it would be worth going over them to have one consistent naming scheme

Reply:
This is due to the icons being imported, they're linked using copied text from FontAwesome.
My standard is camelCase.